src/components/Modal/layout/ModalElements.ts

```diff
@@ -0,0 +1,82 @@
+import styled from 'styled-components';
```

Comment:
Maybe better to rename this one to `styles.tsx`? 
It's done on other components as well.